src/frontend/posts_page/components/SetPublic.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+interface MyProps {
+}
+interface MyState {
+  public: boolean,
+  friendsOnly: boolean,
+  private: boolean
+}
+class SetPublic extends React.Component<MyProps, MyState> {
+  constructor(props: MyProps) {
+    super(props);
```

Comment:
I think this is what Gary meant by the comment of removing `MyProps` because you are not using this interface anyways.
```suggestion
class SetPublic extends React.Component<{}, MyState> {
  constructor(props: {}) {
    super(props);
```

Reply:
Resolved